docs/reference/commandline/service_inspect.md

```diff
@@ -158,6 +158,10 @@ $ docker service inspect --format='{{.Spec.Mode.Replicated.Replicas}}' redis
 10
 ```
 
+#### Restart Policy Delay
+
+The restart policy delay value is reported in `ns`.
```

Comment:
We don't have such a description for the other fields; given that this output is returning the detailed information as returned by the API, it would probably be better to update the information there; https://docs.docker.com/engine/api/v1.40/#operation/ServiceInspect


<img width="784" alt="Screenshot 2020-01-09 at 15 53 29" src="https://user-images.githubusercontent.com/1804568/72077823-36bf3400-32f8-11ea-9096-56ba8b681e75.png">


That documentation is generated from the swagger file in the https://github.com/moby/moby/blob/master/api/swagger.yaml repository.